test/hack/cleanup/main.go

```diff
@@ -71,10 +76,32 @@ func main() {
 
 	// Terminate any old stacks that were provisioned as part of testing
 	// We execute these in serial since we will most likely get rate limited if we try to delete these too aggressively
+	sgInAccount := lo.Must(ec2Client.DescribeSecurityGroups(ctx, &ec2.DescribeSecurityGroupsInput{
```

Comment:
Also, a follow-up question: is this just a back-stop for the security group not deleting as part of the actual test run? Can we make changes so that, in the general case, we delete the security group and successfully delete the VPC but we only use the sweeper if we fail to do so for some other reason